src/ceph_mon.cc

```diff
@@ -727,12 +779,6 @@ int main(int argc, const char **argv)
     prefork.exit(1);
   }
 
-  dout(0) << "starting " << g_conf->name << " rank " << rank
```

Comment:
I removed this since it was duplicative with the log message on line 757 above. not sure I understand why we need two log messages.